application/infrastructure/utils.py

```diff
@@ -0,0 +1,46 @@
+import importlib
+import inspect
+import os
+
+from clapy import Common
+
+
+def get_classes_ending_with(term: str, path_to_search: str):
+    _Classes = []
+
+    for _Root, _Directories, _Files in os.walk(path_to_search):
+
+        DIR_EXCLUSIONS = [r"__pycache__"]
+        FILE_EXCLUSIONS = [r".*__init__\.py", r"^.*(?<!\.py)$"]
+        Common.apply_exclusion_filter(_Directories, DIR_EXCLUSIONS)
+        Common.apply_exclusion_filter(_Files, FILE_EXCLUSIONS)
+
+        for _File in _Files:
+            _Namespace = _Root.replace('/', '.').lstrip(".") + "." + _File[:-3]
```

Comment:
I believe you also need to replace the backslash from the root for windows OS

Reply:
fixed :smile: